### PR TITLE
ui: create step 2 DEX onboarding screens

### LINKED ISSUE #121: [DCRDEX]: create the step 2 onboarding screens
use figma!

Overview page (2)

### PR DESCRIPTION
**Pull request by ukane-philemon**
Closes #121

The add server form on the third step is just a placeholder and the back button on the design is pending feedback from designer @joyce_ofoche

EDIT: Got feedback from @joyce_ofoche and have implemented the back button in https://github.com/crypto-power/cryptopower/pull/135/commits/42e2a99e80f6837e7e7bba2be2ad0acd84b6a44d.

https://github.com/crypto-power/cryptopower/assets/57448127/35f5a887-25bd-4844-8356-da74fa0bd0b5

Designs implemented:

Screenshot 2023-10-02 at 7 02 05 PM

Screenshot 2023-10-02 at 7 01 32 PM

### CODE REVIEW

**Comment by dreacot** [comment]
i also noticed some some comments from wisdom in the design room regarding the dex onboarding page, such as the cert being multi line, having a choose from files option etc.

not sure any conclusion was reached

**Comment by ukane-philemon** [comment]
> such as the cert being multi line, having a choose from files option etc.

Yes, the cert input is multi-line but the file input will wait, I think. More like a v2 design?

**Comment by dreacot** [comment]
> > such as the cert being multi line, having a choose from files option etc.
> 
> Yes, the cert input is multi-line but the file input will wait, I think. More like a v2 design?

okay sounds good. so other concerns raised would be a v2 too?

**Comment by ukane-philemon** [comment]
> so other concerns raised would be a v2 too?

I think the file input is the only thing left for this step(2). The rest would be on the Post Bond page.

**Comment by ukane-philemon** [comment]
UI changes in https://github.com/crypto-power/cryptopower/pull/135/commits/42e2a99e80f6837e7e7bba2be2ad0acd84b6a44d
Screenshot 2023-10-01 at 9 04 54 AM

File: ui/page/dcrdex/dex_onboarding_page.go [hunk 1]
```diff
@@ -185,7 +234,7 @@ func (do *DEXOnboarding) onBoardingStepRow(gtx C) D {
 func (do *DEXOnboarding) onBoardingStep(gtx C, step onboardingStep, stepDesc string) D {
```
Comment (Sirmorrison) [inline ui/page/dcrdex/dex_onboarding_page.go:234]: for uniformity on the app, change `do` to `pg`.

**Comment by Sirmorrison** [comment]
> Closes #121
> 
> The add server form on the third step is just a placeholder and the back button on the design is pending feedback from designer @joyce_ofoche
> 
> EDIT: Got feedback from @joyce_ofoche and have implemented the back button in [42e2a99](https://github.com/crypto-power/cryptopower/commit/42e2a99e80f6837e7e7bba2be2ad0acd84b6a44d). https://github.com/crypto-power/cryptopower/assets/57448127/35f5a887-25bd-4844-8356-da74fa0bd0b5

Please add screenshots to the description on the UI that was implemented.

File: ui/page/dcrdex/dex_onboarding_page.go [hunk 2]
```diff
@@ -240,43 +289,228 @@ func (do *DEXOnboarding) stepSetPassword(gtx C) D {
...
+		}),
+		layout.Rigid(func(gtx layout.Context) layout.Dimensions {
+			return layout.Inset{Top: u16, Bottom: u16}.Layout(gtx, do.nextStepBtn)
+		}),
+	)
+
+	return layoutFlex
+}
+
+// subStepAddServer returns a for to add a server.
```
Comment (dreacot) [inline ui/page/dcrdex/dex_onboarding_page.go:354]:
```suggestion
// subStepAddServer returns a form to add a server.
```

**Review by dreacot — APPROVED** [review]

**Pull request merged**